### PR TITLE
Filter out non-log messages when reading console

### PR DESCRIPTION
**Pull request by m90**
Puppeteer >= 1.4.0 introduced changes that will change the
way messages are passed to `console` events. To filter out
non-test related messages, filter all non-log messages. Also
bump puppeteer to ^1.4.0 and start testing in Node 10.

So this is fixing #171 - looks like we got bitten by: https://github.com/GoogleChrome/puppeteer/pull/2400 which changes what kind of messages will trigger a `console` event.

### CODE REVIEW

File: lib/chromium.js [hunk 1]
```diff
@@ -98,6 +98,9 @@ module.exports = function (b, opts) {
 
       var consumer;
       page.on('console', function (msg) {
+        if (msg.type() !== 'log') {
```
Comment (m90) [inline lib/chromium.js]: @mantoni is this safe to do? Would mochify itself ever use `error`  level?

**Comment by m90** [comment]
I don't fully understand why my npm started locking all the `require`s in the lockfile to be quite honest. Is that something it should do?

**Comment by mantoni** [comment]
I didn't quite get the new review workflow 🙈 I commented a few days back but they don't show until I set a review status and submit them... oh well.

Any non-log console output is not directly printed to stderr. The test cases where changed to distinguish between the two output streams and I've added some tests for `console.log`, `console.info`, `console.warn` and `console.error` to verify.

**Comment by mantoni** [comment]
Shall we release this as a patch or a minor?

**Comment by m90** [comment]
Thanks for adding the tests and `stderr` also seems like a smart thing to do.

Considering that it _also_ adds Node 10, I'd feel it could also be a minor. Should we release it like this?

**Comment by mantoni** [comment]
Sure. Can I ask you to do it? I’m AFK now.

**Pull request merged**